### PR TITLE
RProject.Rtools version 4.2.0.1

### PR DESCRIPTION
**Pull request by dpprdan**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/58751)

### CODE REVIEW

File: manifests/r/RProject/Rtools/4.2.0.1/RProject.Rtools.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: RProject.Rtools
+PackageVersion: 4.2.0.1
+MinimumOSVersion: 10.0.0.0
+Scope: machine
+ProductCode: Rtools42_is1
+Installers:
+- Architecture: x86
```
Comment (ItzLevvie) [inline manifests/r/RProject/Rtools/4.2.0.1/RProject.Rtools.installer.yaml:10]:
```suggestion
- Architecture: x64
```

File: manifests/r/RProject/Rtools/4.2.0.1/RProject.Rtools.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+PackageIdentifier: RProject.Rtools
+PackageVersion: 4.2.0.1
+MinimumOSVersion: 10.0.0.0
+Scope: machine
+ProductCode: Rtools42_is1
+Installers:
+- Architecture: x86
+  InstallerType: inno
+  InstallerUrl: https://cran.r-project.org/bin/windows/Rtools/rtools42/files/rtools42-5168-5107.exe
```
Comment (ItzLevvie) [inline manifests/r/RProject/Rtools/4.2.0.1/RProject.Rtools.installer.yaml:12]:
```suggestion
  InstallerUrl: https://cloud.r-project.org/bin/windows/Rtools/rtools42/files/rtools42-5168-5107.exe
```

File: manifests/r/RProject/Rtools/4.2.0.1/RProject.Rtools.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: RProject.Rtools
+PackageVersion: 4.2.0.1
```
Comment (ItzLevvie) [inline manifests/r/RProject/Rtools/4.2.0.1/RProject.Rtools.installer.yaml:5]:
```suggestion
PackageVersion: 4.2
```
to match Control Panel.

File: manifests/r/RProject/Rtools/4.2.0.1/RProject.Rtools.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: RProject.Rtools
+PackageVersion: 4.2.0.1
```
Comment (ItzLevvie) [inline manifests/r/RProject/Rtools/4.2.0.1/RProject.Rtools.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 4.2
```
to match Control Panel.

File: manifests/r/RProject/Rtools/4.2.0.1/RProject.Rtools.yaml [hunk 5]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: RProject.Rtools
+PackageVersion: 4.2.0.1
```
Comment (ItzLevvie) [inline manifests/r/RProject/Rtools/4.2.0.1/RProject.Rtools.yaml:5]:
```suggestion
PackageVersion: 4.2
```
to match Control Panel.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/r/RProject/Rtools/4.2.0.1/RProject.Rtools.installer.yaml (hunk 1 above)
Comment (dpprdan) [reply manifests/r/RProject/Rtools/4.2.0.1/RProject.Rtools.installer.yaml:10]: 🤦🏻‍♂️

**Comment by SpecterShell** [comment]
The error in the last run:
InstallationVerification-x86-Scope_Undefined-Locale_Undefined-ErrorScreenshot_20220425190930

**Comment by dpprdan** [comment]
> The error in the last run: InstallationVerification-x86-Scope_Undefined-Locale_Undefined-ErrorScreenshot_20220425190930

@SpecterShell Apparently that was run on a 32-bit Windows? Then that would be the proper error message.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**